relay/sources/rudder-package/src/signature.rs

```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// SPDX-FileCopyrightText: 2023 Normation SAS
+
+use anyhow::Result;
+use openpgp::parse::{stream::*, Parse};
+use openpgp::policy::{StandardPolicy, self};
+use openpgp::{Cert, KeyHandle};
+use sequoia_openpgp as openpgp;
+use std::io::{self, Read};
+use crate::SIGNATURE_KEY_PATH;
+
+// This fetches keys and computes the validity of the verification.
+struct Helper {}
+impl VerificationHelper for Helper {
+    fn get_certs(&mut self, _ids: &[KeyHandle]) -> openpgp::Result<Vec<Cert>> {
+
+    // Dependency injection is not possible here, fallback to conditionnal compilation
+    if cfg!(test) {
```

Comment:
I do not understand this part

Reply:
this is absolutely not finished, probably not worth reviewing now